jobs/build.Jenkinsfile

```diff
@@ -458,7 +458,7 @@ lock(resource: "build-${params.STREAM}") {
 
             // parallel build these artifacts
             def pbuilds = [:]
-            ["Aliyun", "AWS", "Azure", "AzureStack", "DigitalOcean", "Exoscale", "GCP", "IBMCloud", "Nutanix", "OpenStack", "VMware", "Vultr"].each {
+            ["Aliyun", "AWS", "Azure", "AzureStack", "DigitalOcean", "Exoscale", "GCP", "IBMCloud", "Nutanix", "OpenStack", "VMware", "virtualbox", "Vultr"].each {
```

Comment:
```suggestion
            ["Aliyun", "AWS", "Azure", "AzureStack", "DigitalOcean", "Exoscale", "GCP", "IBMCloud", "Nutanix", "OpenStack", "VMware", "VirtualBox", "Vultr"].each {
```

??

Reply:
Also, alphabetical order please.

Reply:
Aside: the way we parallelize all these image builds is kinda iffy. We may have already crossed the point where it would've actually been faster to split it into two or more serial groups of parallel runs.